rpcs3/main_application.cpp

```diff
@@ -50,10 +64,60 @@ void main_application::InitializeEmulator(const std::string& user, bool show_gui
 	sys_log.always()("%s", firmware_string);
 }
 
+void main_application::OnEmuSettingsChange()
+{
+	if (Emu.IsRunning())
+	{
+		if (g_cfg.misc.prevent_display_sleep)
+		{
+			enable_display_sleep();
+		}
+		else
+		{
+			disable_display_sleep();
+		}
+	}
+
+	rpcs3::utils::configure_logs();
+
+	if (!Emu.IsStopped())
+	{
+		// Force audio provider
+		if (Emu.IsVsh())
```

Comment:
I'd go with a ternary here to avoid nesting unnecessarily on a trivial bool eval.

Reply:
same here

Reply:
done